lib/api/src/modules/url.ts

```diff
@@ -154,10 +154,6 @@ export const init: ModuleFn = ({ store, navigate, state, provider, fullAPI, ...r
     fullAPI.on(NAVIGATE_URL, (url: string, options: { [k: string]: any }) => {
       fullAPI.navigateUrl(url, options);
     });
-
-    if (fullAPI.showReleaseNotesOnLaunch()) {
-      navigate('/settings/release-notes');
-    }
```

Comment:
moved to the releases module (not deleted)

Reply:
It does feel a bit odd to me that _any_ module can `navigate` when they are initialized. That is why I put this logic here -- to consolidate all of the navigation logic on startup in a place that made sense (the URL module). It is quite messy to jump around and figure out where the navigation is coming from & potentially bug prone since anyone could introduce another navigate call in another module that overrides the release module (as a result of having `init` called afterward). In fact, I spent a lot of time in this PR just figuring out where & why navigation was taking place. What are your thoughts on this topic?

Reply:
There are navigate calls in the stories module as well.